README.md

```diff
@@ -1,3 +1,120 @@
 # Acquia Perz CIS Client for PHP
 
-A PHP Client library to consume the Perz CIS API.
+Provides an API client for PHP applications communicating with the Acquia Personalization Content Index Service .
```

Comment:
let's not confused people with CIS, which is more or an internal name. For the end user, it makes more sense to keep decision and CIS under the same "Personalization API" umbrella.

Reply:
Updated [c2bc3e5](https://github.com/acquia/perz-api-php/pull/6/commits/c2bc3e58cefa96f047f7f92a8626213db95209d4)